.travis.yml

```diff
@@ -15,6 +15,7 @@ install:
 script:
     - pep8 diagnostic_feedback --max-line-length=120
     - pylint diagnostic_feedback --disable=all --enable=function-redefined,undefined-variable,unused-variable
+    - python run_tests.py diagnostic_feedback/tests/integration/*.py
```

Comment:
@jareerahsan This special handling for integration tests should be removed I think. If you make the following files non-executable (`chmod` to `644`), you can remove this line (nose will be able to correctly discover all of the tests):

* diagnostic_feedback/tests/integration/\_\_init\_\_.py
* diagnostic_feedback/tests/integration/base_test.py
* diagnostic_feedback/tests/integration/test_student_bf_quiz.py
* diagnostic_feedback/tests/integration/test_student_dg_quiz.py

While you're at it you could also do the opposite for run_tests.py (i.e., make it executable). That would make it easier to run the tests locally.